scripts/ci/test_plan.py

```diff
@@ -159,12 +162,13 @@ def find_boards(self):
             if p and p.groups():
                 boards.add(p.group(1))
 
+        lb_args = argparse.Namespace(**{ 'arch_roots': [], 'board_roots': [] })
```

Comment:
Is there a reason you chose to leave `board_roots` empty instead of replicating the logic in boards.py to populate it from known modules?

Reply:
Yes. This script only attempt to match file changes in the zephyr repo to board names, so checking for boards matches in other board roots does not make sense.